2-variables/strings.js

```diff
@@ -37,6 +37,42 @@ console.log(userName[2]); // will print "u"
 //🌟ACCESS LAST ENTRY: Since index starts from "0", to access last entry we need to subtract 1 from total string length.
 console.log(userName[userName.length - 1]); // will print "h"
 
+/* 
+  🌟 TEMPLATE LITERALS -
+
+  -> Apart from defining strings in JavaScript using single quotes ('') or double quotes (""), there is a third way to define strings in Javascript too!
+    - We can define strings using backtick quotes (``) as well. For example - `Greetings!`
+    - These type of strings are called as Template Literals or Template strings.
+    - The benefit of using Template literals are that, it allows us to embed variables & expressions inside the string.
+      - This can be done using wrapping the expression inside of ${...} in the string.
+      - The value returned by the expression will then become a part of the string.
+      - TODO - After you learn about the various types of expressions in Javascript, try embedding the various types in template strings to see how they work.
+*/
+
+let userFullName = "Niles Maxwell";
+let numberOfTasksToday = 5;
+
+console.log(`Greetings ${userName}, you have ${numberOfTasksToday} tasks to be done today.`); // Logs "Greetings Niles, you have 5 tasks to be done today."
```

Comment:
Here it's using `userName` variable, so the comment is wrong, it's "Greetings sourabh, ..." You should change the comment and the variable used or remove the variable `userFullName` and change the name in the comment

Reply:
@sabderemane apologies for that. Will fix it right now. Thank you for pointing it out!